x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/table/open_flyout_button.tsx

```diff
@@ -33,18 +29,16 @@ export interface OpenIndicatorFlyoutButtonProps {
 export const OpenIndicatorFlyoutButton: FC<OpenIndicatorFlyoutButtonProps> = ({
   indicator,
   onOpen,
-  isOpen,
 }) => {
   return (
     <EuiToolTip content={VIEW_DETAILS_BUTTON_LABEL} disableScreenReaderOutput>
       <EuiButtonIcon
-        data-test-subj={BUTTON_TEST_ID}
-        color={isOpen ? 'text' : 'primary'}
-        iconType={isOpen ? 'minimize' : 'expand'}
-        isSelected={isOpen}
-        iconSize="s"
         aria-label={VIEW_DETAILS_BUTTON_LABEL}
+        color="text"
+        data-test-subj={BUTTON_TEST_ID}
+        iconType="expand"
         onClick={() => onOpen(indicator)}
```

Comment:
I know it's not strictly in the scope of this PR, but it would be nice if you could replace this with a `useCallback` 😄

Reply:
Same here, this is being completely replaced in the follow up PR [(here](https://github.com/elastic/kibana/pull/231959/files#diff-6227f3edb6de64c09d6689fe0e03845f773325888b49ffd0a67196035982fd23R47)), so this current PR only touched what it really had to...

Reply:
I ended up fixing it anyway as I had to rebase to avoid having the branch outdated